packages/evalite-tests/tests/failing.test.ts

```diff
@@ -1,12 +1,32 @@
 import { createDatabase, getEvalsAsRecord, type Db } from "evalite/db";
 import { runVitest } from "evalite/runner";
-import { expect, it } from "vitest";
+import { expect, it, vitest } from "vitest";
 import { captureStdout, loadFixture } from "./test-utils.js";
 
+it("Should set exitCode to 1 if there is a failing test", async () => {
+  using fixture = loadFixture("failing-test");
+
+  const captured = captureStdout();
+  const exit = vitest.fn();
+  globalThis.process.exit = exit as any;
+
+  await runVitest({
+    cwd: fixture.dir,
+    path: undefined,
+    testOutputWritable: captured.writable,
+    mode: "run-once-and-exit",
+  });
+
+  expect(captured.getOutput()).toContain("failed");
+  expect(exit).toHaveBeenCalledWith(1);
+});
+
 it("Should report a failing test", async () => {
   using fixture = loadFixture("failing-test");
 
   const captured = captureStdout();
+  const exit = vitest.fn();
+  globalThis.process.exit = exit as any;
```

Comment:
I didn't both to repeat the same assertions in these existing tests, but they need to stub exit in order to pass